coordinator/src/db/positions_helper.rs

```diff
@@ -0,0 +1,40 @@
+use crate::db::positions::Position;
+use crate::db::user;
+use crate::notifications::FcmToken;
+use diesel::{Connection, PgConnection, QueryResult};
+use time::OffsetDateTime;
+
+pub fn get_positions_joined_with_fcm_token_with_expiry_within(
+    conn: &mut PgConnection,
+    start: OffsetDateTime,
+    end: OffsetDateTime,
+) -> QueryResult<Vec<(crate::position::models::Position, FcmToken)>> {
+    let result = conn.transaction(|conn| {
+        let users = user::all(conn)?;
+        let positions_with_token =
+            Position::get_all_positions_with_expiry_within(conn, start, end)?
```

Comment:
🔧 The code would be simpler if we'd only consider open positions.

Reply:
Done [`06f1774` (#1419)](https://github.com/get10101/10101/pull/1419/commits/06f17746ccf410ddb5ddb74efd96566117b51cd5)